opencti-platform/opencti-front/src/utils/authorizedMembers.ts

```diff
@@ -54,8 +55,9 @@ export const authorizedMembersToOptions = (
     });
 };
 
-export const getCurrentUserAccessRight = (userAccessRight: string | null | undefined) => {
-  const canManage = userAccessRight === 'admin';
+export const useGetCurrentUserAccessRight = (userAccessRight: string | null | undefined) => {
+  const isBypass = useGranted([BYPASS]);
```

Comment:
why ?

Reply:
A bypass user is considered as canManage of everything, why does it seem to be a problem ?